datasets/at/pep/at_pep.yaml

```
@@ -0,0 +1,29 @@
+title: Austria's Politicians 
+entry_point: crawler.py
+prefix: at-pep
+coverage:
+  frequency: daily
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  List of politicians and others officials from Austria.
+description: |
+  This dataset includes continuously updated dossiers of
+  all members of the National Council, members of the Federal
+  Government and the Bundesrat as well as all state governments
+  and state parliaments.  Other dossiers concern
+  the Federal President, the general secretaries of the
+  ministries, the Ombudsmen and many more.
```

```suggestion
  This dataset identifies
  all members of the National Council, members of the Federal
  Government and the Bundesrat as well as all state governments
  and state parliaments.  Other significant posts in this dataset include
  the Federal President, the general secretaries of the
  ministries, the Ombudsmen and many more.
```

The source website has dossiers, or perhaps dossier is a bad google translation from the German - we just have some cursory information here. So I don't think `dossier` quite fits.